### DISCUSSION TITLE
Enabling text selection

### DISCUSSION THREAD

**Discussion by daniel-keller** [thegetty/quire#115] [Get Help]
Our curators want to enable text selection/text copying in our catalog. 
Is there a config option I'm missing to do this?

**Comment by geealbers** [comment]
@daniel-keller there's not config option, so you'll need to do it in css. The fact that text selection wasn't enabled was actually an unintentional part of earlier versions of the default theme, we've since changed it so text can always be selected. Would be interested to hear if the Quire community actually would like the option though.

**Reply by ronvoluted** [reply]
A scenario where that might be desirable would be decorative text. One example, though admittedly rare:
![image](https://user-images.githubusercontent.com/5785323/120268555-5e49de80-c2e9-11eb-8e66-24409dd16e4a.png)
